PortalFetch/crawler_download.py

```diff
@@ -204,6 +206,38 @@ def waitUtilPageLoaded(driver, count):
     raise ElementNotVisibleException("Could not load the full page!")
 
 
+def generateFilenameAndQuarter(quarterValue):
+    """Return quarter and fileName.
+
+    Args:
+        quarterValue:the quarter_value in crawler.config
+    Returns:
+        quarter str and fileName str
+
+    """
+    year = quarterValue[0:4]
+    resultList = []
+    quarterSwitcher = {
+        "1": "Summer",
+        "2": "Fall",
+        "3": "Winter",
+        "4": "Spring",
+    }
+    schoolSwitcher = {
+        "1": "Foothill",
+        "2": "De Anza",
+    }
+    school = schoolSwitcher.get(quarterValue[5], "")
+    quarter = quarterSwitcher.get(quarterValue[4], "")
+    qr = year + " " + quarter + " " + school
+    resultList.append(qr)
+    if school == "De Anza":
+        school = "De_Anza"
+    fn = year + "_" + quarter + "_" + school + "_courseData.json"
+    resultList.append(fn)
+    return resultList
```

Comment:
the option value in the dropdown list is a bit tricky, they use summer as the start of a new academic year. This implies that the summer quarter of 202x is actually the first quarter of 202(x+1), so you will need to change some code logic here so that if the input value is ```202112```, the filename and first key inside should correspond to summer 2020 instead of summer 2021